pydra/engine/tests/test_shelltask.py

```diff
@@ -320,3 +321,30 @@ def test_wf_shell_cmd_1(plugin):
     res = wf.result()
     assert "_result.pklz" in res.output.out
     assert "_task.pklz" in res.output.out
+
+
+# tests with new OutputSpec
+
+
+def test_shell_cmd_outputspec_1(tmpdir):
+    """
+        customised OutputSpec
+    """
+    cmd = ["touch"]
+    args = ["newfile_tmp.txt"]
+
+    @dc.dataclass
+    class MyOutput(ShellOutSpec):
+        newfile: str = "newfile_tmp.txt"
```

Comment:
should be of type `File`

Reply:
could change this, but in `task` I assumed that this is a relative path and I'm adding `self.output_dir`